operations/research-and-development/product/development-process/mobile-feature-guidelines.md

```diff
@@ -0,0 +1,39 @@
+# Mobile Feature Guidelines 
+
+New product features should consider the mobile client at the same level of importance as the webapp client. For new features implemented on the server or webapp, please consider the mobile client through these lenses:
+
+## #1 Are mobile app code changes needed?
+
+- A good rule of thumb is that if there are webapp repo code changes (outside of the system console UI), mobile code changes are needed to support the feature. However, not every webapp feature needs parity with mobile, but that is a discussion to have with **PM & UX** and a decision to be made in the requirements gathering phase of feature design.
+- If a feature requires mobile code changes, the mobile team PM (Eric Sethna) should be made aware of the feature **before** it’s committed to the feature teams quarterly roadmap. Please involve the mobile team PM in roadmap planning and quarterly OKR planning discussions. The same applies even if the feature team will be doing the mobile implementation themselves, given that the mobile team will be involved in code reviews, testing and should provide guidance on the feature implementation.
+- During design and spec development of any feature that will involve mobile code changes, please involve the mobile PM (Eric Sethna) and mobile dev lead (Elias Nahum) at all stakeholder checkpoints  including requirements gathering, design option exploration, and final design review. 
+
+
+## #2 Backwards Compatibility
+
+- Mobile releases must be backwards compatible with at least all server versions back to the [oldest supported ESR](https://docs.mattermost.com/administration/extended-support-release.html?highlight=esr). Question to ask during design: What is the expected behaviour on a new server with an old app, or a new app with an old server? Example: 
+  - New mobile app with an old server: If the server is running older code and does not support the new feature, we can’t allow users to access the feature from a new mobile app. 
+    - Example: Hide the “Mark as Unread” option if the server is older than v5.18
+      - [Use the isMinimumServerVersion helper function](https://github.com/mattermost/mattermost-mobile/blob/master/app/screens/post_options/index.js#L49)
+  - New server with an old mobile app: Users will not be able access the new features from their mobile device since it’s running older code, but the new server code should not cause regressions to a users experience on an older mobile app.
```

Comment:
```suggestion
  - New server with an old mobile app: Users will not be able access the new features from their mobile device since it’s running older code, but the new server code should not cause regressions to a user's experience on an older mobile app.
```